index.d.ts

```diff
@@ -188,6 +188,24 @@ export interface ThemeSectionProperties {
 
 export type ThemePropertyType = 'color' | 'font' | 'backgroundSelector' | 'backgroundColor';
 
+export type Orientation = 'landscape' | 'portrait';
+
+export type WidgetType = 'main' | 'banner' | 'ticker';
+
+export interface Layout {
+  orientation: Orientation;
+  widgetType?: WidgetType;
+  zones?: number;
+  position?: number;
+  flip?: boolean;
+}
+
+export interface FeedData {
+  socialItems?: Array<any>;
+  socialFeeds?: Array<any>;
+  trigger: any;
```

Comment:
Is it possible to specify the `trigger` type?

Reply:
added generic type